sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/README.md

```diff
@@ -10,7 +10,7 @@ This library follows the [new Azure SDK guidelines](https://azure.github.io/azur
     - Better error-handling.
     - Support uniform telemetry across all languages.
 
-## Getting started 
+## Getting started
 
 ### Install the package
 
```

Comment:
You need to remove the `--prerelease` option in the sample below, if you want to release an official version.